projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/payment-method/payment-method.component.ts

```diff
@@ -26,7 +26,7 @@ export class PaymentMethodComponent implements OnInit {
   newPaymentFormManuallyOpened = false;
   existingPaymentMethods$: Observable<any>;
```

Comment:
`any` can be `PaymentDetails[]`. Please also update `any` inside `user.service.ts`. I found there are many `any` in it.

Reply:
Also, there are some auth leftovers, which should be update once the `user.service.ts` is done:
* `login.component.ts` has `user$: Observable<any>;`
* `register.component.ts` has `titles$: Observable<any>;`